gateway/ln-gateway/src/lib.rs

```diff
@@ -486,6 +488,22 @@ impl Gateway {
         }
     }
 
+    pub fn gateway_id(&self) -> PublicKey {
+        self.gateway_id
+    }
+
+    pub fn versioned_api(&self) -> &SafeUrl {
+        &self.versioned_api
+    }
+
+    pub async fn clone_gateway_config(&self) -> Option<GatewayConfiguration> {
+        self.gateway_config.read().await.clone()
+    }
+
+    pub async fn clone_state(&self) -> GatewayState {
```

Comment:
I kind of don't see a point of `clone_` here. This type seems so cheap, that we might just call it `get_state`.

Reply:
Done.